app/models/handlebar_type.rb

```diff
@@ -2,7 +2,7 @@ class HandlebarType
   include Enumable
 
   SLUGS = {
-    drop: 5,
+    drop_bar: 5,
```

Comment:
`drop` conflicts with a built-in method definition, so this renames the slug to `drop_bar`. Not wedded to it if another term would be preferable